endpoints/openrtb2/auction.go

```diff
@@ -205,6 +205,8 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 
 	activityControl = privacy.NewActivityControl(&account.Privacy)
 
+	hookExecutor.SetActivityControl(activityControl)
```

Comment:
Do we need to call this for other endpoints as well?

Reply:
Ooof, yes, I will add this to amp and video.

Reply:
Added to AMP endpoint only. 
In Video endpoint hooks are not configured: 
`HookExecutor:          hookexecution.EmptyHookExecutor{},`